fsw/src/ds_msg.h

```diff
@@ -356,6 +356,22 @@ typedef struct
     DS_FileInfo_t FileInfo[DS_DEST_FILE_CNT]; /**< \brief Current state of destination files */
 } DS_FileInfoPkt_t;
 
+/**
+ * \brief Completed application file info
+ */
+typedef struct
+{
+    CFE_MSG_TelemetryHeader_t TlmHeader; /**< \brief cFE Software Bus telemetry message header */
+
+    DS_FileInfo_t FileInfo; /**< \brief Current state of destination file */
+} DS_FileCompletePkt_t;
+
+typedef union
+{
+    CFE_SB_Buffer_t      SBBuf;
+    DS_FileCompletePkt_t Pkt;
+} DS_FileCompletePktBuf_t;
```

Comment:
Add doxygen documentation (this typedef supports CFE SB Allocate buffer use with a DS_FileCompletePkt_t that matches the alignment constraints of a SBBuf)